gulpfile.js/tasks/lint.js

```diff
@@ -0,0 +1,36 @@
+const gulp = require('gulp');
+const gulpUtil = require('gulp-util');
+const minimist = require('minimist');
+const spawn = require('child_process').spawn;
+const stylelint = require('stylelint');
+
+/**
+ * Lints the CSS for errors.
+ */
+function _lintCss() {
+  // Pass all command line flags to EsLint.
+  const args = minimist( process.argv.slice( 2 ) );
+  const willFix = args.fix;
+  let willFixFlag = '';
+
+  if (willFix) {
+    willFixFlag = '--fix';
+  }
+
+  return new Promise( ( resolve, reject ) => {
+    spawn(
+      'node_modules/stylelint/bin/stylelint.js',
+      ['client/**/*.scss', 'wagtail/**/static_src/**/scss/**/*.scss', willFixFlag],
+      { stdio: 'inherit' }
+    ).once( 'close', code => {
+      if ( code ) {
+        gulpUtil.log( 'Stylelint exited with code ' + code );
+        reject();
+      }
+      gulpUtil.log( 'Stylelint done!' );
+      resolve();
```

Comment:
Can you explain why you chose to integrate stylelint with gulp? As far as I can see this isn't reading files from the  input of another vinyl stream but from a hard-coded list. Couldn't we have the same result by directly calling the stylelint CLI from `npm run lint:css`, like we do for scss_lint?

Reply:
I think the errors being wrapped within npm's is ok. Just updated the `make lint` script so it silences them there at least.